rebar.config.script

```diff
@@ -157,7 +157,8 @@ DepDescs = [
 {jiffy,            "jiffy",            {tag, "CouchDB-1.0.9-2"}},
 {mochiweb,         "mochiweb",         {tag, "v3.2.1"}},
 {meck,             "meck",             {tag, "0.9.2"}},
-{recon,            "recon",            {tag, "2.5.3"}}
+{recon,            "recon",            {tag, "2.5.3"}},
+{fast_pbkdf2,      {url, "https://github.com/esl/fast_pbkdf2"}, {tag, "1.0.2"}}
```

Comment:
Since we're planning a CouchDB release soon-ish, wonder if it makes sense to just vendor it for now?

At some point https://github.com/erlang/otp/pull/7770 will merge then if we made a release with https://github.com/esl/fast_pbkdf2's mirror we'd have to archive it and keep it essentially forever.

Reply:
by "just vendor it" you mean "keep pointing at the /esl/ repo we don't control"? I don't think it's an option per ASF rules.

Reply:
No I meant include it in the applications directory alongside ets_lru, exxhash and others.

Reply:
hmmmm interesting